### PR TITLE
chore: add keywords

### PR DESCRIPTION
**Pull request by antbern**
## Motivation and Context

Adds keywords to the entire workspace which is then inherited by the wasm binding crate and copied over to the `package.json` by `wasm-pack` (tested).

Please add more keywords via suggestions! 🔥 

## Checklist

Please ensure that your PR meets the following requirements:

### General

- [ ] Code follows project style and guidelines.
- [ ] No redundant or duplicate code.
- [ ] No added new dependencies without clear justification and approval.

### Documentation

- [ ] Added/updated relevant documentation (e.g., README, inline comments).
- [ ] Added a CHANGELOG entry for major changes.

### Testing

- [ ] Added relevant test cases, leveraging tools like `rstest` or similar for reusable tests.
- [ ] Tests pass locally.

### Build & CI/CD

- [ ] Confirmed no CI/CD pipeline issues.
- [ ] Updated build scripts where necessary.

## Comments to Reviewer

Please provide any comments or points of consideration for the reviewers, such as specific areas of focus, potential edge cases, or context that would aid the review process.

### Reviewer Responsibilities

- [ ] Code adheres to style and guidelines.
- [ ] All tests are appropriately covered and they pass.
- [ ] Consider performance and/or security impacts.
- [ ] Constructive feedback is provided.

### CODE REVIEW

**Review by hauju — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -27,14 +27,17 @@ license = "AGPL-3.0-only"
 license-file = "LICENSE"
 homepage = "https://www.etogruppe.com/en/information/legal-notice.html"
 repository = "https://github.com/ETOSPHERES-Labs/cawaena-sdk"
+keywords = ["wallet", "web3", "payments", "crypto", "kyc"]
```
Comment (kristifidani) [inline Cargo.toml:30]: ERC20, EVM, blockchain, etopay, etospheres,

Comment (kristifidani) [reply Cargo.toml:30]: maybe we should put these inside the wasm since we are talking about pushing to crates.io at that allows only 5 keywords. While npm might allow more

Comment (hauju) [reply Cargo.toml:30]: Yeah lets, focusing on 5 keywords for now.

Comment (antbern) [reply Cargo.toml]: Added 5 keywords for the crates and all of them to the wasm crate (since npm does not have limit). What do you think?

Comment (kristifidani) [reply Cargo.toml]: Looks good for now 👍

**Review by kristifidani — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (antbern) [reply Cargo.toml]: Also added some categories!
https://crates.io/category_slugs

**Review by kristifidani — APPROVED** [review]
